test/source/TestString.inl

```diff
@@ -1163,7 +1163,7 @@ int TEST_STRING_NAME()
 		typename StringType::value_type buf[64];
 
 		StringType str(LITERAL("abcdefghijklmnopqrstuvwxyz"));
-		str.copy(buf, 10, sizeof buf);
+		str.copy(buf, 10, 0);
```

Comment:
Pretty sure that's still wrong, as the verify statement has a 9 character literal but compares 10 characters.